k8s/src/main/scala/io/buoyant/k8s/Watchable.scala

```diff
@@ -22,10 +22,6 @@ private[k8s] abstract class Watchable[O <: KubeObject: TypeReference, W <: Watch
 
   protected def backoffs: Stream[Duration]
   protected def stats: StatsReceiver
-  // whether or not to pass resource versions on watches; this should be true
-  // for List resources & false for object resources.
-  // TODO: there is probably a more elegant way to represent this...
-  protected val watchResourceVersion: Boolean
```

Comment:
I think that all of the changes in this file and k8s/resources.scala need to be reverted -- they contain reverts of changes that you made in the `eliza/object-watchable`.